Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+pipeline {
+    agent {
+        docker {
+            image 'teambitflow/golang-build:1.12-alpine'
+            args '-v /root/.goroot:/go'
+        }
+    }
+    stages {
+        stage('Build & test') { 
+            steps {
+                    sh 'go clean -i -v ./...'
+                    sh 'go install -v ./...'
+                    sh 'rm -rf reports && mkdir -p reports'
+                    sh 'go test -v ./... 2>&1 | go-junit-report > reports/test.xml'
+                    sh 'go vet ./... &> reports/vet.txt'
+                    sh 'golint $(go list -f "{{.Dir}}" ./...) &> reports/lint.txt'
```

Comment:
Do you want to use the lint report somewhere?

Reply:
Yes, the SonarQube Golang Scanner has the ability to parse the go vet and
golint outputs :) I prepared them for it, but for now they are just
archived. I don't know yet how test coverage works.

Reply:
BTW, I found that runnin each command in its individual 'sh' is very convenient in the BlueOcean view, as the output of every command is then captured and displayed individually.

Reply:
True, maybe I will change the other Jenkinsfile as well